how-to/kubeflow_assets.md

```diff
@@ -28,10 +28,7 @@ https://github.com/kubeflow-bot
 
 Admins
 
-* chrisheecho@kubeflow.org
 * ewj@kubeflow.org
-* jlewi@kubeflow.org
-* sarahmaddox@kubeflow.org
 * thea@kubeflow.org
```

Comment:
I think Yuan is a GSuite admin. /cc @Bobgy

Reply:
Yes, and I added both @james-jwu and @paveldournov .